docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+services:
+    mldonkey:
+        stdin_open: true
```

Comment:
Not sure why you added this. Should not be necessary at all. Did you add it for a specific reason?